src/uct/ib/base/ib_md.c

```diff
@@ -282,13 +287,22 @@ uct_ib_md_print_mem_reg_err_msg(const char *title, void *address, size_t length,
     ucs_log(level, "%s", ucs_string_buffer_cstr(&msg));
 }
 
+static void uct_ib_dereg_mrs(struct ibv_mr **mrs, int count)
```

Comment:
always inline?

Reply:
this is unrelate to my PR, bad rebase, pushed a revert commit @iyastreb